jcache/embedded/src/main/java/org/infinispan/jcache/annotation/InjectedCacheResolver.java

```diff
@@ -35,36 +39,55 @@
 
    private EmbeddedCacheManager defaultCacheManager;
 
-   private final Map<EmbeddedCacheManager, JCacheManager> jcacheManagers =
-         new HashMap<EmbeddedCacheManager, JCacheManager>();
+   private final Map<EmbeddedCacheManager, JCacheManager> jcacheManagers = new HashMap<>();
    private JCacheManager defaultJCacheManager;
 
+   private EmbeddedCacheManager getDefaultCacheManager() {
+      if (defaultCacheManager == null) {
+         defaultCacheManager = getBeanReference(EmbeddedCacheManager.class);
+      }
+      return defaultCacheManager;
+   }
+
+   private JCacheManager getDefaultJCacheManager() {
```

Comment:
This method is unnecessary, just reference defaultCacheManager, which is private. Same with setter.